### PR TITLE
Updated Polish translation.

### LINKED ISSUE #2656: Translate "Open in new private tab" and more words in options
**Is your feature request related to a problem? Please describe.**
Is there a option to translate this ? Also in Options some words is not translated.

**Describe the solution you'd like**
Update polish translation, translate all untranslated words. I can translate but can't find where i can do that.

**Describe alternatives you've considered**
Another way to translate it manually, via userchrome.css or something...

**Additional context**
screen:
https://i.imgur.com/ayJEeLZ.png

### PR DESCRIPTION
**Pull request by kamil445**
Updated Polish translation.

Also I added extensibles.ftl file mentioned here https://github.com/WaterfoxCo/Waterfox/pull/2662 to translate more words.

### CODE REVIEW

File: browser/locales/l10n/pl/browser/browser/extensibles.ftl [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# In browser/locales/jar.mn
+# Localised versions MUST be located in browser/locales/l10n/{locale}/browser/browser/extensibles.ftl
+
+open-all-private =
+    .label = Otwórz wszytkie w prywatnych kartach
```
Comment (hawkeye116477) [inline browser/locales/l10n/pl/browser/browser/extensibles.ftl]: You eaten one letter.

(wszy **s** tkie)

It's little more of that, so search and replace all :smile:.

**Review by adamp01 — APPROVED** [review]
lgtm - will merge after https://github.com/WaterfoxCo/Waterfox/pull/2662

**Comment by adamp01** [comment]
@kamil445 - We are going to be adding another translation today:
```
### aboutDialog
update-updateCheckDisabled = Checking for updates is currently disabled.
```

**Review by hawkeye116477 — CHANGES_REQUESTED** [review]

**Comment by kamil445** [comment]
> @kamil445 - We are going to be adding another translation today:
> 
> ```
> ### aboutDialog
> update-updateCheckDisabled = Checking for updates is currently disabled.
> ```

Thank you for information, i've added translation.

File: browser/locales/l10n/pl/browser/browser/extensibles.ftl (hunk 1 above)
Comment (kamil445) [reply browser/locales/l10n/pl/browser/browser/extensibles.ftl]: Thanks, is corrected now :)

**Review by hawkeye116477 — APPROVED** [review]

**Comment by MrAlex94** [comment]
Sorry for the delay!

This is now handled by Crowdin and will be live in G5 Beta 2. 

435bd9a2683e09600655dc322098650479a51a5a

**Pull request closed without merging**